app/routes/questions/$question.tsx

```diff
@@ -33,11 +33,13 @@ export function Question({
   onLazyLoadQuestion,
   onToggle,
   selectQuestion,
+  ...props
 }: {
   questionProps: Question
   onLazyLoadQuestion: (question: Question) => void
   onToggle: ReturnType<typeof useQuestionStateInUrl>['toggleQuestion']
   selectQuestion: (pageid: string, title: string) => void
+  [k: string]: JSX.IntrinsicElements['div']
```

Comment:
this is used for `article` element,, but I guess there is no difference in type of props, so probably fine to keep